app/models/metrics.rb

```diff
@@ -13,15 +13,16 @@ class << self
       613, # Calls to graph_url_engagement_count have exceeded the rate of 10 calls per 3600 seconds
     ]
 
-    NUMBER_OF_DAYS_TO_UPDATE = 9
+    # we are going to update once in 2 days
+    NUMBER_OF_DAYS_TO_UPDATE = 1
```

Comment:
Noting that since it's only 1, we don't need a constant and the `times` loop below, but on the other hand, let's keep some flexibility, so it's fine to keep the block.

Reply:
Yup. Thought the same thing, since it's not clear what the issue with metrics is, we might want to increase it again in a not so distant future.

Reply:
Yes exactly!